web/src/components/ContrastThemeToggle.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { ReactElement } from 'react'
+import { useTranslation } from 'react-i18next'
+import styled from 'styled-components'
+
+import { ContrastIcon } from '../assets'
+import { useContrastTheme } from '../hooks/useContrastTheme'
+import useWindowDimensions from '../hooks/useWindowDimensions'
+import ToolbarItem from './ToolbarItem'
+import Button from './base/Button'
+import Icon from './base/Icon'
+
+const ContrastButton = styled(Button)`
+  display: flex;
+  width: 100%;
+  padding: 24px 0;
+  align-items: center;
+  cursor: pointer;
+
+  & > span {
+    padding: 0 28px;
+    color: ${props => props.theme.colors.textColor};
+  }
+`
+
+const StyledIcon = styled(Icon)<{ $isContrastTheme: 'light' | 'contrast' }>`
+  width: 24px;
+  height: 24px;
+  transform: ${({ $isContrastTheme }) => ($isContrastTheme === 'contrast' ? 'reverse' : 'none')};
```

Comment:
🔧 This does not use the `reverse` property of the icon. Please have a look e.g. `TtsPlayer` and search for `reverse`.

Reply:
:see_no_evil: I thought there is a property called 'reverse' and visually I have not noticed it

Reply:
You should use the `reverse` property as follows:
``` tsx
const StyledIcon = styled(Icon)`
  transition: transform 0.2s ease;
`
...
        <StyledIcon reverse={themeType === 'contrast'} src={ContrastIcon} />
```

Reply:
Actually, you are only reversing the icon for the kebab menu at the moment, right? I think you don't do that for the toolbar item. I think you should either reverse it for both cases or not at all. I think without reversing it actually looks more "reversed" than with.

Reply:
yes, you are right. Because of color switch, it gives the feeling as if it is switching. So I would just remove it as you suggest :+1: